### DISCUSSION TITLE
网站上的图片不显示了怎么办

### DISCUSSION THREAD

**Discussion by Zikyaai99** [tangly1024/NotionNext#1340] [Q&A]
在Notion笔记上显示没有问题，
![图片](https://github.com/tangly1024/NotionNext/assets/134773687/c81d88ee-1c11-4e68-b9a9-90023d3586eb)
当时在网站中就不显示了
![图片](https://github.com/tangly1024/NotionNext/assets/134773687/c17daf74-5f0b-4816-8607-1ef2760ea980)
求解答

**Comment by aqiandev** [comment]
me too
